### ISSUE TITLE
Bad inlined-markdown rendering in changelog

### ISSUE THREAD

**Issue by isoos** [dart-lang/pub-dev#6273]
The recent pana release has `Fix: revert the extra line in <summary> blocks.` in the changelog.md, and it is rendered as `<summary> blocks.</summary>`. (It is similar on GitHub too).

I think we should not allow such inlined HTML elements, and should render them as text instead.

**Comment by satvik2131** [comment]
Hi @isoos , I would like to work on this issue , can I start implementing ?

**Comment by isoos** [comment]
@satvik2131: What kind of fix do you have in mind? I think this may need to be fixed in `package:markdown` as a generic verification for inlined HTML.

**Comment by satvik2131** [comment]
we can use code highlighting , I have also updated the file , please review it .

**Comment by isoos** [comment]
Yeah, that's a good thing going forward, however, the version that was published already has the markup (and will show up in analysis for a while).

**Comment by sigurdm** [comment]
Seems to be working now.
